functions.php

```diff
@@ -50,6 +50,10 @@ function course_maker_localization_setup() {
 // Add the custom meta boxes.
 require_once CHILD_THEME_DIR . '/lib/metaboxes.php';
 
+// Add the the attachment meta box class.
+require_once CHILD_THEME_DIR . '/lib/meta-attachments.php';
+require_once CHILD_THEME_DIR . '/lib/meta-attachments-init.php';
+
```

Comment:
I have the main class file, which contains the featured article files. One is a class, and the other an inisitalizer.